_posts/2015-01-27-assume-the-user-is-correct.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Assume the user is correct"
+layout: posts
+location: "Washington, DC"
+author: Dave Copeland
+author_url: 'http://naildrivin5.com'
+image: "hizzy.jpg"
+published: true
+---
+
+A mentioned [previously][internal-apps-post], a lot of what we do at Stitch Fix is create software to *run* Stitch Fix.
+While we have a team dedicated to our (external) customers' experience, most engineers are working on software for internal users.
+
+[internal-apps-post]: http://technology.stitchfix.com/blog/2014/08/27/tip-of-the-iceberg/
+
+We realized a key part of our success would be a good relationship with the internal users.
```

Comment:
Seems like a tense change between the first sentence and the other two? "A key part of our success is a good relationship with our internal users"?